CppCoreGuidelines.md

```diff
@@ -19181,6 +19182,33 @@ A header should include all its dependencies. Be careful about using relative pa
 
 A test should verify that the header file itself compiles or that a cpp file which only includes the header file compiles.
 
+### <a name="Rs-incscope"></a>SF.12: Prefer the angle bracket form of `#include` where you can and the quoted form everywhere else
+
+##### Reason
+
+The [standard](http://eel.is/c++draft/cpp.include) provides flexability for compilers to implement
+the two forms of `#include` selected using the angle (`<>`) or quoted (`""`) syntax. Vendors take
+advantage of this and and use different search algorythms and methods for specifying the include path.
+
+Neveer the less, the gudance is to use the angle form when possible. This supports the fact that the
+standard library headers must be includedthis way, is more likely to create portable code, and enables
+the quoted form for other uses. For example being clear about the locality of the header relative
+to files that includes it or in scearios where the different search algorythm is required.
+
+##### Example
+
+    #include <string>       // From the standard library, required form
+    #include "helpers.h"    // A project specific file, use "" form
+
+##### Note
+Failing to follow this results in difficult to diagnose errors due to picking up the wrong file by incorrectly specifying the scope when it is included.
+
+Library creators should put their headers in a folder and have clients include those files using the relative path `#include <some_library/common.h>`
+
+##### Enforcement
+
+A test should verify that the headers referenced via `<>` are from the standard library and those using `""` are not.
```

Comment:
Doesn't that contradict the rule to use angle brackets where possible?

Reply:
I missed updating this, this was from the 2nd version.